docs/2.0/integration/testng.adoc

```diff
@@ -167,23 +167,36 @@ They can: be parametrized, make checks, have nested steps, and create attachment
 
 In order to define steps in Java code, you need to annotate the respective methods with **@Step** annotation.
 When not specified, the step name is equal to the annotated method name converted to human-readable format.
-To define an explicit step name, pass your own **String** argument:
+
+Note that steps' mechanics was revised and now it supports smart fields' analysis.
```

Comment:
Good, that is exactly what I was thinking about. IMHO it's better to state explicitly that there is no "old way" support anymore and customers will have to refactor their existing code upon migration.